{"uuid":"c78a3d1e-003b-4556-8ae3-da232fd2c20c","name":"Scenario Outline: The user getting error message with invalid python code for Types of Linked List","children":["fc7be0e4-8f6b-4d3e-9ec4-58244fdc243d"],"befores":[],"afters":[{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696561156008,"stop":1696561156009},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696561156085,"stop":1696561156085},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696561156189,"stop":1696561156189},{"name":"Hooks.Hooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696561158234,"stop":1696561158234}],"start":1696561155704,"stop":1696561158235}